internal/configs/virtualserver.go

```diff
@@ -630,18 +644,26 @@ func createUpstreamServersForPlus(virtualServerEx *VirtualServerEx) map[string][
 	virtualServerUpstreamNamer := newUpstreamNamerForVirtualServer(virtualServerEx.VirtualServer)
 
 	for _, u := range virtualServerEx.VirtualServer.Spec.Upstreams {
+		if _, isExternalNameSvc := virtualServerEx.ExternalNameSvcs[u.Service]; isExternalNameSvc {
```

Comment:
i think it makes sense to add a case to `TestCreateUpstreamServersForPlus` or new test function for VS/VSR and externalname svc.